### DISCUSSION TITLE
how to disable promoted label?

### DISCUSSION THREAD

**Discussion by xsmyqf** [TriliumNext/Trilium#3228] [Q&A] [answered]
I have a label like this:
`#label:Tag=promoted,multi,text`
Is there a way to disable it? I only want to use it display something.

**Comment by zadam** [comment, marked as answer]
What do you mean by "disable it"? You can simply remove it if you don't want these to be promoted.

**Reply by xsmyqf** [reply]
I want to display something there only, not editable.

Is there a way to add a custom promoted attribute type like date , text ? I mean using html5 to make a custom attribute. I can't find some guiding note to do it.

**Reply by zadam** [reply]
I see, there isn't a way to do that. But I also don't understand the need - the editing interface allows you to both read and update the value. Since Trilium is single user app, I don't see a lot of value in forbidding yourself to update it.

**Reply by xsmyqf** [reply]
except for some specific usage, I think making the attribute as a custom component which can be easily extended is a good idea. someone can use html5 to make any promoted attribute.
